deploy/helm/sumologic/README.md

```diff
@@ -20,6 +20,7 @@ The following table lists the configurable parameters of the Sumo Logic chart an
+| `sumologic.metrics.provider`                                              | Set provider to use for metrics forwarding and metadata enrichment. Can be either otelcol or fluentd.                                                                                                                                                                                                                                                                                                                                         | `fluentd`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
```

Comment:
Can we make this `sumologic.metrics.metadata.provider`?
So that we don't confuse people as this does not provide metrics, only enrich them with metadata.

Reply:
we can